package.json

```diff
@@ -6,7 +6,21 @@
   "keywords": [
     "bitfinex"
   ],
-  "dependencies": {
+  "scripts": {
+    "test": "npm run lint && npm run test:code",
+    "test:code": "NODE_ENV=test mocha ./test",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix ."
+  },
+  "devDependencies": {
+    "mocha-it-each": "^1.0.2",
```

Comment:
Mocha dependency is missing.
Maybe you have mocha installed globally.

Reply:
good catch